src/MSALWrapper/AuthFlow/AuthFlowFactory.cs

```diff
@@ -45,6 +45,13 @@ public class AuthFlowFactory
             // as it sets the order in which auth flows will be attempted.
             List<IAuthFlow> flows = new List<IAuthFlow>();
 
+            // We try IWA as the first auth flow as it works for any Windows version
+            // and tries to auth silently if it finds an account in the cache.

```

Comment:
nit: IWA is a silent flow, and doesn't work off of the MSAL cache, but rather your Windows login session state.

Reply:
I was referring to trying `AcquireTokenSilent` if we find account in cache. If account is null, then we try IWA. Let me know if I should state this a bit differently. :)

Reply:
I think fine for now, we ack the opporuntity to refactor the MSAL cache silent attempt into it's own AuthFlow.

Reply:
Perfect, bug has been logged to track this.